engine/src/main/java/org/hibernate/search/analyzer/impl/LuceneEmbeddedAnalyzerStrategy.java

```diff
@@ -35,7 +35,7 @@
  */
 public class LuceneEmbeddedAnalyzerStrategy implements AnalyzerStrategy {
 
-	private static final Log log = LoggerFactory.make();
+	private static final Log LOG = LoggerFactory.make();
```

Comment:
I have no strong opinion, for the sake of sharing my view:

- the new LOG ones are the ones you created, I simply didn't complain before but they looked odd ;-)
- I definitely prefer smaller PRs when possible, as my main problem is not to find 3 hours to review PRs, but to find 3 contiguous uninterrupted hours.